embedding-calculator/src/services/facescan/plugins/facenet/facemask/facemask.py

```diff
@@ -0,0 +1,64 @@
+#  Copyright (c) 2020 the original author or authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from typing import Tuple, Union
+
+import numpy as np
+import tensorflow as tf2
+from tensorflow.keras.models import load_model
+from cached_property import cached_property
+
+from src.services.imgtools.types import Array3D
+from src.services.facescan.plugins import base
+from src.services.dto import plugin_result
+
+import cv2
+
+
+class MaskDetector(base.BasePlugin):
+    slug = 'mask'
+    LABELS = ('without_mask', 'with_mask', 'mask_weared_incorrect')
+    ml_models = (
+        ('face_mask_detector', '1AeSYb_E_3cqZM67qXnJ__wJDgw6yqTDV'),
```

Comment:
Name of the model architecture and probably dataset, instead of 'face_mask_detector'